src/SDK.ts

```diff
@@ -413,16 +415,16 @@ export default class SDK implements ISDK {
    * @param requestId: RequestId to use for session init.
    * @param queueAvailabilityOptionalParams: Optional parameters for session init.
    */
-  public async getQueueAvailability(requestId: string, queueAvailabilityOptionalParams: IGetQueueAvailabilityOptionalParams = {}): Promise<QueueAvailability> {
+  public async getAgentAvailability(requestId: string, queueAvailabilityOptionalParams: IGetQueueAvailabilityOptionalParams = {}): Promise<QueueAvailability> {
```

Comment:
Can we rename the interfaces `queueAvailabilityOptionalParams`, `IGetQueueAvailabilityOptionalParams` & `QueueAvailability` accordingly?